src/enhancers/omit/OmitProps.ts

```diff
@@ -0,0 +1,9 @@
+import { omit } from 'lodash/fp';
+import { compose, mapProps } from 'recompose';
+import { Omit } from '../../types/Omit';
+
+export const omitProps = <TProps, TOmitted>(keys: Array<keyof TProps>) =>
```

Comment:
Is this used?

Reply:
No, I was gonna use it. Then I switched theming solution.
But I thought that it is a nice enhancer to have, since recompose doesn't include one.